Assets/Fungus/Thirdparty/FungusLua/Scripts/ConversationManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+namespace Fungus
+{
+	class ConversationManager : MonoBehaviour
```

Comment:
ConversationManager could be a normal c# class, instantiated and initialized from LuaUtils. Would be more encapsulated and less likely to have unexpected side effects than a singleton monobehavior.